app/utility/transforms/CollapsedMetrics.js

```diff
@@ -262,6 +262,32 @@ export const getMaxDuration = (set) => {
     return getMaxMetrics(durations);
 };
 
+// Peak-End
+
+export const getPeakEnd = (set) => {
+    const min = getMinAvgVelocity(set);
+    const velocities = getAvgVelocities(set);
+    const lastRepVelocity = velocities[velocities.length - 1];
```

Comment:
- [ ] you need to check for the length before you get the last rep velocity. THAT or actually use getLastRepVelocity if we have it